device-code-flow/app.ps1

```diff
@@ -33,19 +33,56 @@ public static class DeviceCodeHelper
 }
 "@ -ReferencedAssemblies $RequiredAssemblies -IgnoreWarnings -WarningAction SilentlyContinue
 
+# Include the MsalCacheHelper used for token caching
+if (-not ('Microsoft.Identity.Client.Extensions.Msal.MsalCacheHelper' -as [type])) {
+    try {
+        Add-Type -Path (Join-Path (Split-Path $PSCommandPath) 'Microsoft.Identity.Client.Extensions.Msal.2.19.6\lib\netcoreapp2.1\Microsoft.Identity.Client.Extensions.Msal.dll')
+    }
+    catch {
+        Write-Error $_
+        return
+    }
+}
 
 # 'Application (client) ID' of app registration in Azure portal - this value is a GUID
 $ClientId = ""
 
 # 'Directory (tenant) ID' of app registration in Azure portal - this value is a GUID
 $TenantId = ""
 
+# Set to 0 by default.  Set this value to 1 to clear the cache.
+$clearCache = 1
```

Comment:
Is this an instruction for the user?  "Set to 0 by default" or a statement of fact? Not sure there is enough context on this setting for a user to know what to do with this.  If it's 0 by default, why is it 1 here?  Can you add some clarity?

Reply:
This can get tossed.  It is more so related to disk-based caching solely for example purposes.